src/components/ebay-drawer/component.js

```diff
@@ -0,0 +1,69 @@
+const findIndex = require('core-js-pure/features/array/find-index');
```

Comment:
Looks like this PR contains the `drawer` stuff as well? I'm assuming this shouldn't be the case.

Reply:
Yes it is. This branch is cut off from the drawer branch because i needed some changes from it and it was not yet merged
Once the drawer is merged it should only have the toast changes.